packages/components/package.json

```diff
@@ -45,7 +45,7 @@
     "alpheios-client-adapters": "^3.3.0",
     "alpheios-data-models": "^3.3.0",
     "alpheios-experience": "github:alpheios-project/experience",
-    "alpheios-fixtures": "github:alpheios-project/fixtures",
+    "alpheios-fixtures": "github:alpheios-project/fixtures#i75-add-ti/s",
```

Comment:
I believe, it would be removed

Reply:
oh, shoot I thought I caught all of these. Thanks, I will fix it!